.jenkins

```diff
@@ -3,12 +3,11 @@
 pipeline {
 
   agent {
-   label 'golang-next'
+    label 'golang-next'
   }
 
   environment {
     IMAGE_NAME = 'vaporio/emulator-plugin'
-    GOVERSION = '1.12'
```

Comment:
yeah, this gets set in the dockerfile. But its different, we set GOLANG_VERSION vs GOVERSION

https://github.com/vapor-ware/golang/blob/master/1.13/bionic/Dockerfile#L12